CHANGELOG.md

```diff
@@ -22,6 +22,9 @@
 
 #### Enhancements
 
+* Adds `empty_string` opt-in rule.  
```

Comment:
Please add your entry to the **Master** section of the change log. Version 0.25.0's already out.